contrib/internal/httputil/trace.go

```diff
@@ -28,6 +28,7 @@ type TraceConfig struct {
 	QueryParams    bool                      // specifies that request query parameters should be appended to http.url tag
 	FinishOpts     []ddtrace.FinishOption    // span finish options to be applied
 	SpanOpts       []ddtrace.StartSpanOption // additional span options to be applied
+	AppSecParams   httpsec.AppSecParams      // extra parameters that AppSec may need while wrapping the request handler
```

Comment:
Related to my other comment, I think we can at the moment simplify this to:
```suggestion
	PathParams     map[string]string         // specifies path parameters of the handler, monitored by appsec
```